src/plot/axis/axis-ticks.js

```diff
@@ -171,9 +173,17 @@ class AxisTicks extends React.Component {
       const text = tickFormatFn(v);
 
       return (
-        <g key={i} {...translateFn(pos, 0)} className="rv-xy-plot__axis__tick">
-          <line {...pathProps} className="rv-xy-plot__axis__tick__line"/>
-          <text {...textProps} className="rv-xy-plot__axis__tick__text">
+        <g key={i} {...translateFn(pos, 0)}
+        className="rv-xy-plot__axis__tick"
+        style={style}>
+          <line {...pathProps}
+            className="rv-xy-plot__axis__tick__line"
+            style={{...style, ...style.line}}
```

Comment:
This could potentially make for weird results if you use an `em` based measurement in the base style object.